sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -302,6 +329,20 @@ def __repr__(self):
             :1024
         ]
 
+    def to_dict(self):
+        value = self.value
+        if isinstance(self.value, dict):
+            value = {k: v.to_dict() for k, v in self.value}
```

Comment:
```suggestion
            value = {k: v.to_dict() for k, v in self.value.items()}
```

Reply:
Done, also added the real world example test.